[bugs] add check for LP: #1906280

freyes opened the pull request:
This patch implements a check to detect ocurrences of bug 1906280, this
will affect systems running systemd>=237-3ubuntu10.43 and
charm-neutron-openvswitch<21.01

Related-Bug: https://bugs.launchpad.net/charm-ovn-chassis/+bug/1906280

dosaboy commented on tests/unit/fake_data_root/sos_commands/dpkg/dpkg_-l:
```diff
@@ -947,7 +947,7 @@ ii  ssh-import-id                          5.5-0ubuntu1
 ii  sysstat                                11.2.0-1ubuntu0.3                               amd64        system performance tools for Linux
-ii  systemd                                229-4ubuntu21.28                                amd64        system and service manager
+ii  systemd                                237-3ubuntu10.46                                amd64        system and service manager
 ii  systemd-sysv                           229-4ubuntu21.28                                amd64        system and service manager - SysV links
```
can you make all the systemd* packages the same version

freyes replied:
done.

dosaboy reviewed (changes requested):
Hi Felipe, I would like to land this asap but its gonna need two small updates; first can you fix the systemd version (see comment) and it also needs a rebase since there are some imports that have changed name. Thanks.

freyes commented:
> Hi Felipe, I would like to land this asap but its gonna need two small updates; first can you fix the systemd version (see comment) and it also needs a rebase since there are some imports that have changed name. Thanks.

Updated the fixture, rebased, resolved merge conflicts, tox is happy ;-)

dosaboy reviewed (changes requested):
Hey freyes, I am hitting an error when i run a snap contaning this change:

-Traceback (most recent call last):
  File "/snap/hotsos/x1/plugins/openstack/parts/octavia_bugs.py", line 8, in <module>
    import git
ModuleNotFoundError: No module named 'git'

We either need to suppress that exception or update the snapcraft.yaml to install that dep

freyes commented:
Added a fix for this issue, which it's basically make the snap to pull
the dependencies listed in the requirements.txt

On Wed, 2021-06-02 at 02:48 -0700, Edward Hope-Morley wrote:
> @dosaboy requested changes on this pull request.
> Hey freyes, I am hitting an error when i run a snap contaning this
> change:
> -Traceback (most recent call last):
> File "/snap/hotsos/x1/plugins/openstack/parts/octavia_bugs.py", line
> 8, in 
> import git
> ModuleNotFoundError: No module named 'git'
> We either need to suppress that exception or update the
> snapcraft.yaml to install that dep
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or unsubscribe.

dosaboy commented:
closing for now. will revisit later.
